### DISCUSSION TITLE
Renovate bot stopped creating PRs

### DISCUSSION THREAD

**Discussion by just-jeb** [renovatebot/renovate#17249] [Request Help]
I have renovate configured here: https://github.com/just-jeb/angular-builders and it worked fine, however, at this moment the Renovate Bot hasn't been creating PRs for almost a month.  
I tried reinstalling the Github App with no luck. Any insights on what might go wrong with that? Thanks! 🙏🏿

**Comment by HonkingGoose** [comment]
**EDIT:** The edit history for the Dependency Dashboard shows Renovate ran three days ago. So it's not "stuck".

---

Try clicking this checkbox in your Dependency Dashboard:

> - [ ] Check this box to trigger a request for Renovate to run again on this repository

Maybe that will get Renovate to run again on your repository? 😉

**Reply by HonkingGoose** [reply]
Found the problem! You set the maximum number of Renovate PRs to 10 with the `:prConcurrentLimit10` preset: 

https://github.com/just-jeb/angular-builders/blob/d4ecb42fbfb025080427dcd6fdc03087a95d6d84/renovate.json#L4

If you want more PRs you have these options:

- Eat your vegetables and merge some update PRs, or
- Replace [`:prConcurrentLimit10`](https://docs.renovatebot.com/presets-default/#prconcurrentlimit10) with [`:prConcurrentLimit20`](https://docs.renovatebot.com/presets-default/#prconcurrentlimit20) in your extends array

---

**Edit:** you can probably auto-merge Jest updates, Jasmine updates, `@types` packages, and linting packages like `prettier` or `eslint` once the tests pass. I recommend you read the [Renovate docs, key concepts automerge](https://docs.renovatebot.com/key-concepts/automerge/).

**Reply by just-jeb** [reply]
Oh, thank you so much! I'll check it out!

**Reply by viceice** [reply]
or close renovate PR's you don't like to merge soon
